tests/query/atomWithQuery.test.tsx

```diff
@@ -269,15 +269,14 @@ it('query with enabled (#500)', async () => {
   await findByText('hidden')
 
   fireEvent.click(getByText('toggle'))
-  await findByText('loading')
```

Comment:
@dai-shi this was the issue previously -- because we are remounting the same query atom, the previous value in the cache is used immediately, thus there is no suspend.

I'll work to improve this test in a follow up PR related to `enabled` functionality